terra/tests/utils.py

```diff
@@ -87,17 +116,29 @@ def tearDown(self):
 
 
 class TestComputeCase(TestCase):
+  '''
+  Test case that mocks for ``_connection`` in ``terra.compute.utils.compute``.
+  This allows for a compute to be retrieved from the ``ComputeHandler`` for a
+  single test. More useful when used in :class:`TestLoggerConfigureCase`
```

Comment:
```suggestion
  Test case that allows for a compute to be retrieved from the
  ``ComputeHandler`` for a single test. This is done by mocking ``_connection``
  in ``terra.compute.utils.compute``, so that the Handler does not stay
  initialized. More useful when used in :class:`TestLoggerConfigureCase`.
```